src/devtools/views/layout.js

```diff
@@ -24,9 +24,11 @@ class Layout extends React.Component {
       return null
     }
 
+    let { history } = this.props
+
     return (
       <div className={css.rail}>
-        <ActionList history={this.props.history} />
+        <ActionList focused={history.focused} history={history} />
```

Comment:
I think you can revert the changes here since you're pulling `focused` out of `history` in your `ActionList` anyway.